src/Addon/FieldType/FieldTypeQuery.php

```diff
@@ -71,10 +71,14 @@ public function filter(Builder $query, FilterInterface $filter)
             $query->{$this->where()}(
                 function (Builder $query) use ($translations, $filter, $column) {
 
-                    $query->where(
-                        "{$translations}.locale",
-                        config('app.locale')
-                    );
+                    /**
+                     *  closed this block because it breaks 'multiple field type'.
```

Comment:
why it's cause error? PR to original repo? @muammertop

Reply:
Repeats in multiple_fieldtype and keeps it not working @fatihalp @spektra2147